lib/WooCommerce/Subscription.php

```diff
@@ -148,44 +148,59 @@ public function subscribeOnCheckout($orderId, $data) {
     $subscriber = Subscriber::where('email', $data['billing_email'])
       ->where('is_woocommerce_user', 1)
       ->findOne();
+
     if (!$subscriber) {
       // no subscriber: WooCommerce sync didn't work
       return null;
     }
 
-    $checkoutOptinEnabled = (bool)$this->settings->get(self::OPTIN_ENABLED_SETTING_NAME);
+    $checkoutOptinEnabled = (bool)$this->settings->get( self::OPTIN_ENABLED_SETTING_NAME );
```

Comment:
Could you please remove the extra spaces? It is multiple times in this file. I thought our PHP coding standard checker is configured better.